docs/acme-divergences.md

```diff
@@ -59,6 +59,13 @@ Boulder does not implement applications, instead it implements the `new-cert` fl
 
 Boulder does not implement the `reason` field for the `revoke-cert` endpoint, `unspecified` (0) from [RFC3280 Section 5.3.1](https://tools.ietf.org/html/rfc3280#section-5.3.1) is used for all requests.
 
+## [Section 6.6.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.6)
+
+Boulder considers the following keys authorized to revoke a certificate:
+
+1. The account key that initially created the certificate being revoked
+2. The public key in the certificate being revoked
```

Comment:
Good idea. I changed the wording slightly to match the other sections and added this in 9095a8ca6f0797ecb41b6eaf4f6f848c6a9db858